Jamulus.pro

```diff
@@ -94,19 +94,19 @@ win32 {
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
         contains(QT_ARCH, "i386") {
-        	message("Native i386 build")
-        	libjackname = libjack.lib
+            message("Native i386 build")
+            libjackname = libjack.lib
         } else {
-        	message("Native x86_64 build")
-		    libjackname = libjack64.lib
-		    !exists("C:/Program Files (86)/JACK2/include/jack/jack.h") {
-            	message("Warning: jack.h was not found at the usual place, install JACK2 32 bits to cross compile.")
-         	}	
+            message("Native x86_64 build")
+            libjackname = libjack64.lib
+            !exists("C:/Program Files (86)/JACK2/include/jack/jack.h") {
+                message("Warning: jack.h was not found at the usual place, install JACK2 32 bits to cross compile.")
+            }
         }
           !exists("C:/Program Files/JACK2/include/jack/jack.h") {
-         	 message("Warning: jack.h was not found at the usual place, maybe JACK2 64 bits to compile")
+            message("Warning: jack.h was not found at the usual place, maybe JACK2 64 bits to compile")
```

Comment:
This message is not correct. This shows for both cases (x86_64 and i386). But when you build on a 32-bit platform, you should install the 32-bit JACK version. Perhaps extend the text with this information? Something like "install JACK2 32/64 bits depending on your OS architecture to compile"
Also the text "maybe JACK2" would be more accurate if it is "install JACK2".

Reply:
Modified in 02db193